namespaces/live-1.cloud-platform.service.justice.gov.uk/court-probation-preprod/07-certificate.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: cert-manager.io/v1alpha3
+kind: Certificate
+metadata:
+  name: court-probation-preprod-cert
+  namespace: court-probation-prod
```

Comment:
Should the namespace name read: `court-probation-preprod`?

Reply:
Yes it should 🤦🏻‍♀️